plugins/node/instrumentation-tedious/package.json

```diff
@@ -47,20 +47,23 @@
     "@opentelemetry/api": "^1.3.0"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "6.21.0",
```

Comment:
[v6 is still compatible with eslint v8 and adds support for es2022 as well.](https://github.com/typescript-eslint/typescript-eslint/pull/8377)